### PR TITLE
chore(apis): Update ownership from issues to alerts & notifications

### PR DESCRIPTION
**Pull request by sentaur-athena**
After re-orgs some endpoints are better owned by alerts & notifications vs. issues. I'm not sure if the ownership I'm assigning in this PR is correct so will get help from both teams.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -174,6 +174,15 @@ yarn.lock                                                @getsentry/owners-js-de
 /tests/snuba/incidents/                                   @getsentry/alerts-notifications
 /tests/sentry/snuba/test_query_subscription_consumer.py   @getsentry/alerts-notifications
 /tests/sentry/snuba/test_subscriptions.py                 @getsentry/alerts-notifications
+/src/sentry/api/endpoints/notification_defaults.py                          @getsentry/alerts-notifications
+/src/sentry/api/endpoints/broadcast_details.py                              @getsentry/alerts-notifications
+/src/sentry/api/endpoints/broadcast_index.py                                @getsentry/alerts-notifications
```
Comment (saponifi3d) [inline .github/CODEOWNERS:179]: sounds like we don't own either of these

File: src/sentry/api/endpoints/user_subscriptions.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ class NewsletterValidator(serializers.Serializer):
 
 @control_silo_endpoint
 class UserSubscriptionsEndpoint(UserEndpoint):
-    owner = ApiOwner.ISSUES
+    owner = ApiOwner.ALERTS_NOTIFICATIONS
```
Comment (saponifi3d) [inline src/sentry/api/endpoints/user_subscriptions.py:29]: not sure if this should be us or billing? i'm a little overloaded on the term `subscriptions` 😅

**Review by mrduncan — APPROVED** [review]
Issues specific changes LGTM - thank you for the cleanup!

**Review by vartec — APPROVED** [review]
TY!

File: src/sentry/api/endpoints/user_subscriptions.py (hunk 2 above)
Comment (sentaur-athena) [reply src/sentry/api/endpoints/user_subscriptions.py:29]: yeah no, it's the emails people get. It's more like marketing but there's no eng team supporting it. I'll add it to unowned.

**Pull request merged**